device/celestica/x86_64-cel_silverstone-r0/led_proc_init.soc

```diff
@@ -0,0 +1,8 @@
+#The Port LED of Silverstone SONIC can't work well, after the issue is fixed by SAI, The led will start. 
+#led auto on; led start
+sleep 10
```

Comment:
this sleep is not needed.

Reply:
removed in https://github.com/Azure/sonic-buildimage/pull/5424/commits/d6b6a8d6097226443937aace14cc14ad327308f9